README.md

```diff
@@ -1,28 +1,60 @@
 # NumPyro 
 
 [![Build Status](https://travis-ci.com/pyro-ppl/numpyro.svg?branch=master)](https://travis-ci.com/pyro-ppl/numpyro)
+[![Documentation Status](https://readthedocs.org/projects/numpyro/badge/?version=latest)](https://numpyro.readthedocs.io/en/latest/?badge=latest)
 
-[Pyro](https://github.com/pyro-ppl/pyro) on Numpy. This uses 
-[JAX](https://github.com/google/jax) for autograd and JIT support. This is an 
-early stage experimental library that is under active development, and there are 
-likely to be  many changes to the API and internal classes, as the design evolves. 
+Probabilistic programming with Numpy on GPUs powered by [JAX](https://github.com/google/jax) for autograd and JIT support. 
```

Comment:
nit: I guess you mean CPU/GPU? (and maybe TPU when JAX supports TPU on the cloud).

Reply:
I thought it will be good to keep the summary concise, since running on CPUs is almost always supported, but I will clarify it below. I didn't want to get into TPUs because that's more a focus for the JAX team, but if we have success running NumPyro on TPUs, we should highlight that later. 😄

Reply:
yup, agree. I tried to use TPU in colab this morning but none of the installation methods in jax/README works. Then I realize that (also mentioned in the README) jax team is working on supporting cloud TPU. So running on TPU might not be possible at the current stage. :D

Reply:
Clarified the summary, and I think all your comments should be addressed. :)